src/signals/incident-management/containers/IncidentDetail/components/StatusForm/components/EmailPreview/EmailPreview.tsx

```diff
@@ -0,0 +1,73 @@
+// SPDX-License-Identifier: MPL-2.0
+// Copyright (C) 2022 Gemeente Amsterdam
+import type { FC } from 'react'
+import styled from 'styled-components'
+import Button from 'components/Button'
+import { themeColor, themeSpacing } from '@amsterdam/asc-ui'
+import ModalHeader from '../ModalHeader/ModalHeader'
+
+const ButtonWrapper = styled.div`
+  box-sizing: border-box;
+  padding: ${themeSpacing(4)};
+  border-top: 1px solid ${themeColor('tint', 'level4')};
+`
+
+const ModalContainer = styled.div`
+  display: flex;
+  flex-direction: column;
+`
+
+const StyledButton = styled(Button)`
+  height: ${themeSpacing(11)};
+  margin-right: ${themeSpacing(2)};
+`
+
+const StyledIframe = styled.iframe`
+  border: none;
+  padding-left: ${themeSpacing(2)};
+`
+
+interface EmailPreviewProps {
+  emailBody: string
+  onClose: () => void
+  onUpdate: () => void
+}
+
+const fontStyling = '<style>*{font-family:Avenir Next;}</style>'
+
+const EmailPreview: FC<EmailPreviewProps> = ({
+  emailBody,
+  onUpdate,
+  onClose,
+}) => {
+  return (
+    <ModalContainer>
+      <ModalHeader title="Controleer bericht aan melder" onClose={onClose} />
+      <StyledIframe
+        data-testid="emailBodyIframe"
+        srcDoc={emailBody.concat(fontStyling)}
+        height={'500vh'}
+      />
+      <ButtonWrapper>
```

Comment:
There is a `FormFooter` component that might be used instead. It's also used in the `FilterForm` component as wel as other components.

Reply:
👍